Source/Particles/PhysicalParticleContainer.H

```diff
@@ -203,6 +203,12 @@ public:
                          amrex::Real x_rms, amrex::Real y_rms, amrex::Real z_rms,
                          amrex::Real q_tot, long npart, int do_symmetrize);
 
+    /** Load a particle beam from an external file
+     *
+     * @param[in] s_f path to an external file in openPMD markup
+     */
+    void AddExternalFileBeam(const std::string s_f);
```

Comment:
"Beam" can be confusing as it is sometimes used for a laser beam too. Besides, it seems that this capability can be used to input any plasma profile, it doesn't have to be a beam. What about renaming this function `AddPlasmaFromFile`? I think that would be consistent with our `AddPlasma` naming. This would also require changes in the cpp file.

Reply:
Thanks for the suggestion @MaxThevenet 
That is a great point. I totally agree! I will work on changing all the "beam" entries.